### PR TITLE
JSAssignment 3 : ECMA Script Features

### PR DESCRIPTION
**Pull request by swatipatil-webonise**
Problem Statement : 
1) Write a ES code to print numbers by adding delay.
2) Write a ES code to print delay in ES using closure.

### CODE REVIEW

File: assignment3/src/problem1_UsingES6.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>ES Demo</title>
+        <script>
+            function printNumbers() {
+                const WAIT_TIME = 1000;
+                for (let iterator = 1; iterator < 10; iterator++) {
+                    function timeout(iterator) {
```
Comment (deepakb-webonise) [inline assignment3/src/problem1_UsingES6.html]: No need to create the function here.

File: assignment3/src/problem1_UsingES6.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                        setTimeout(() => document.write(iterator + '<br>'), iterator * WAIT_TIME);
+                    }
+                    timeout(iterator);
```
Comment (deepakb-webonise) [inline assignment3/src/problem1_UsingES6.html]: In the case where we define and function and use it, we can go with an anonymous function.

File: assignment3/src/problem1_UsingJS5.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Closure Demo</title>
+        <script>
+            function printNumbers() {
+                const SLEEP_TIME = 1000;
+                let counter = 1;
+                setInterval(function () {
```
Comment (deepakb-webonise) [inline assignment3/src/problem1_UsingJS5.html]: setInterval should be cleared. The program doesn't stop after printing 10 numbers.

Comment (swapnil-webonise) [inline assignment3/src/problem1_UsingJS5.html]: Prefer arrow function

File: assignment3/src/problem1_UsingJS5.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+                    document.write(counter++ + '<br>');
```
Comment (swapnil-webonise) [inline assignment3/src/problem1_UsingJS5.html]: we can write like

document.write(`${counter + 1} <br>`);

**Review by swapnil-webonise — CHANGES_REQUESTED** [review]

**Review by swapnil-webonise — APPROVED** [review]

**Comment by swapnil-webonise** [comment]
@deepakb-webonise Merge PR if fine from your end.

**Review by deepakb-webonise — APPROVED** [review]

**Pull request merged**